### ISSUE TITLE
Resistor values

### ISSUE THREAD

**Issue by thetechknight** [ppieczul/macintosh-portable#1]
Did you read these resistor values from a properly working Hybrid? or was it an unknown one? 

Because if not, then chances are those resistor values are going to be wrong, and will need cross-checked on a working example that has zero issue. The trouble with these is the Hybrid ICs go bad because the capacitor goo gets into the resistors and change their values, throwing all the setpoints off.

**Comment by ppieczul** [comment]
I don’t know of the hybryd board was working. The whole motherboard was not operational.

**Comment by thetechknight** [comment]
Yeah, I didnt figure. So the resistor values are probably incorrect. 

I will have to go back and double-verify.
